homeassistant/helpers/config_entry_oauth2_flow.py

```diff
@@ -209,7 +209,10 @@ async def _token_request(self, data: dict) -> dict:
             error_code = error_response.get("error", "unknown")
             error_description = error_response.get("error_description", "unknown error")
             _LOGGER.error(
-                "Token request failed (%s): %s", error_code, error_description
+                "Token request to %s failed (%s): %s",
+                self.token_url,
```

Comment:
What I don't like is that the URL is printed first, error code/description last. The user might just skip over that.

I also don't know if the token URL is that important. It's usually bad redirect URI or client ID/secret.

Reply:
It's a way to pinpoint which integration this is causing it. My own prod currently struggles from the message shown in the description and I currently don't know which integration exactly is causing this. And there is a debug logger above this, so I can know this by enabling debug logging for `homeassistant.helpers.config_entry_oauth2_flow`, but that's an extra step. As in, when helping people, being able to pinpoint the integration in one logging entry is quite valuable.

Reply:
How about using `self.domain` ?
The URL is an implementation detail that the user generally does not know about.
I think the domain is generally more recognizable.

Reply:
Oh haven't checked if that's available

Reply:
I think the latest change to use the domain settles the comment.